docs/Commands.md

```diff
@@ -2,11 +2,130 @@
+
+### `/block` & `/unblock`
+
+Usage: `/(block|unblock) <user>` 
+
+Blocks or unblocks the specified user. Blocking will hide that user's messages/whispers as well as disassociate them from you on Twitch. For more information about blocking, see [Twitch's help article](https://help.twitch.tv/s/article/how-to-manage-harassment-in-chat?language=en_US#BlockWhispersandMessagesfromStrangers).
+
+### `/chatters`
+
+Usage: `/chatters`
+
+Shows the total amount of chatters currently connected to the channel.
+
+### `/clearmessages`
+
+Usage: `/clearmessages`
+
+Clears all messages from the current split/channel. This command is purely visual and is not related to the `/clear` moderator command. 
+
+### `/clip`
+
+Usage: `/clip`
+
+Creates a clip from the last 30 seconds of the current channel. Also provides an edit link to edit a clip from the current point in time.
+
+### `/debug-args`
+
+Usage: `/debug-args`
+
+Displays the arguments that chatterino was launched with.
+
+### `/delete`
+
+Usage: `/delete <msg-id>`
+
+Moderator only, deletes a message in chat with the provided ID. Usually used in mod actions as they can pass through the message ID.
+
+### `/fakemsg`
+
+Usage: `/fakemsg <raw message>`
+
+Displays the provided IRC messsage in chat as if it was sent from Twitch's IRC server.
+### `/marker`
+
+Usage: `/marker`
+
+[Editor](https://help.twitch.tv/s/article/Managing-Roles-for-your-Channel#types) only, creates a [stream marker](https://help.twitch.tv/s/article/creating-highlights-and-stream-markers?language=en_US#markers) in the current stream. Streamer must be live to create markers.
+
+### `/openurl`
+
+Usage: `/openurl`
+
+Opens a URL in the default web browser. Useful in custom commands.
+
+### `/popout`
+
+Usage: `/popout [channel]`
+
+Opens the current channel or provided channel's chat in Twitch's popout webchat, using your default browser.
+
+### `/popup`
+
+Usage: `/popup [channel]`
+
+Opens the current channel or provided channel's chat in a new window.
+
+### `/r`
+
+Usage: `/r <message>`
+
+Replies to the last received whisper.
+
+### `/setgame`
+
+Usage: `/setgame <game>`
+
+Broadcaster only, sets the current channel's game to the best match of the provided game name.
+
+### `/settitle`
+
+Usage: `/settitle <title>`
+
+Broadcaster only, sets the current channel's title to the provided text.
+
+### `/streamlink`
+
+Usage: `/streamlink [channel]`
+
+Attempts to open current or specified stream in streamlink. For more information, see [Streamlink's website](https://streamlink.github.io/).
+
+### `/uptime`
+
+Usage: `/uptime`
+
+Displays the uptime of the current channel's livestream.
+
+### `/user`
+
+Usage: `/user <user> [channel]`
+
+Opens the specified user's Twitch usercard for the given channel (or the channel provided) in the default browser.
+
+### `/usercard`
+
+Usage: `/user <user> [channel]`
+
+Opens the specified user's Chatterino usercard for the given channel or the channel provided.
+
+### `/w (Whisper)`
```

Comment:
I believe (Whisper) should be outside of the backticks, since it isn't apart of the command itself.

Reply:
Moved.